web-client/extras/lib/NumberUtils.ts

```diff
@@ -0,0 +1,42 @@
+export class NumberUtils {
+    static UINT8_MAX = 255;
+    static UINT16_MAX = 65535;
+    static UINT32_MAX = 4294967295;
+    static UINT64_MAX = Number.MAX_SAFE_INTEGER;
+
+    static isInteger(val: unknown): val is number {
+        return Number.isInteger(val);
+    }
+
+    static isUint8(val: unknown): boolean {
+        return NumberUtils.isInteger(val)
+            && val >= 0 && val <= NumberUtils.UINT8_MAX;
+    }
+
+    static isUint16(val: unknown): boolean {
+        return NumberUtils.isInteger(val)
+            && val >= 0 && val <= NumberUtils.UINT16_MAX;
+    }
+
+    static isUint32(val: unknown): boolean {
+        return NumberUtils.isInteger(val)
+            && val >= 0 && val <= NumberUtils.UINT32_MAX;
+    }
+
+    static isUint64(val: unknown): boolean {
+        return NumberUtils.isInteger(val)
+            && val >= 0 && val <= NumberUtils.UINT64_MAX;
+    }
+
+    static randomUint32(): number {
+        return Math.floor(Math.random() * (NumberUtils.UINT32_MAX + 1));
+    }
+
+    static randomUint64(): number {
+        return Math.floor(Math.random() * (NumberUtils.UINT64_MAX + 1));
+    }
```

Comment:
This doesn't reach every 64-bit integer due to the float precision. Probably unused and can be removed anyway.

Reply:
Addressed in #2975